server/command.go

```diff
@@ -466,6 +484,35 @@ func startSubscriptionMigration(p *Plugin, context *model.CommandArgs, args ...s
 	return &model.CommandResponse{}
 }
 
+func deleteOldSubscription(p *Plugin, context *model.CommandArgs, args ...string) *model.CommandResponse {
+	if !utils.IsSystemAdmin(context.UserId) {
+		p.postCommandResponse(context, installOnlySystemAdmin)
+		return &model.CommandResponse{}
+	}
+
+	oldSubscriptions, getErr := service.GetOldSubscriptions()
+	if getErr != nil {
+		p.postCommandResponse(context, getErr.Error())
+		return &model.CommandResponse{}
+	}
+
+	if len(oldSubscriptions) == 0 {
+		p.postCommandResponse(context, NoOldSubscriptionDeleteMsg)
+		return &model.CommandResponse{}
+	}
+
+	go func() {
+		if err := p.API.KVDelete(storePackage.GetOldSubscriptionKey()); err != nil {
+			p.API.LogError("Unable to delete old subscriptions", "Error", err.Error())
+		}
```

Comment:
shouldn't we return from here?

Reply:
no no in the error case we are just loggin the error in server logs and all the processes will be there as it is